metro.transform.js

```diff
@@ -15,7 +15,9 @@ module.exports.transform = function applyRainbowTransform({
           '@react-native-community/clipboard': {},
           'react-native-keychain': {},
           'react-native-video': {},
-          'react-native-webview': {},
+          'react-native-webview': {
+            exceptions: ['@ratio.me/ratio-react-native-library'],
```

Comment:
This is new post this https://github.com/rainbow-me/metro-plugin-anisotropic-transform/pull/1

Reply:
nice